src/Framework/Hosting.AspNetCore/Hosting/AspNetCoreDotvvmRequestContextExtensions.cs

```diff
@@ -35,5 +35,14 @@ public static AuthenticationManager GetAuthentication(this IDotvvmRequestContext
             }
             return new AuthenticationManager(concreteContext.OriginalContext);
         }
+
+
+        /// <summary>
+        /// Gets the <see cref="IDotvvmRequestContext"/> bound to the specified <see cref="HttpContext"/>.
+        /// </summary>
+        public static IDotvvmRequestContext? GetDotvvmContext(this HttpContext httpContext)
+        {
```

Comment:
:thinking: We forgot to add null annotations to this package. Anyway, we should add it later, not in this PR

Reply:
Removed the annotation, we can do the refactoring later